go.mod

```diff
@@ -93,7 +93,7 @@ require (
 	github.com/mrunalp/fileutils v0.0.0-20160930181131-4ee1cc9a8058 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20120707110453-a547fc61f48d
 	github.com/mvdan/xurls v1.1.0
-	github.com/onsi/ginkgo v1.8.0
+	github.com/onsi/ginkgo v1.10.1
 	github.com/onsi/gomega v1.5.0
```

Comment:
Shouldn't this also be updated to 1.7.0 if you're making this change?

Reply:
It could get updated, but it's unrelated to the problem that this PR is solving and both projects are only loosely related, so Ginkgo 1.10.1 and Gomega 1.5.0 should be a valid combination.

Considering that both the Ginkgo and the Gomega update could cause issues, it might be safer to update them separately.

Reply:
The releases we're clearly coordinated when looking at the commit history, please update gomega to ensure we are in lock step.

Reply:
/hold

Reply:
> The releases we're clearly coordinated when looking at the commit history, please update gomega to ensure we are in lock step.

Okay, if that's preferred.

Reply:
/hold cancel

Gomega updated, too. I also rebased.